ci/Jenkinsfile_utils.groovy

```diff
@@ -57,20 +57,20 @@ def init_git_win() {
 }
 
 // pack libraries for later use
-def pack_lib(name, libs) {
+def pack_lib(name, libs=mx_lib) {
```

Comment:
Unrelated changes?

Reply:
Oops, it was a careless copy and paste. Now I removed the default value.